internal/pithosctl/pkg/pithos/keys.go

```diff
@@ -31,19 +31,18 @@ const (
 	secretKeyLength = 20
 )
 
-func generateAccessKey(tenant string, master bool) (*cluster.AccessKey, error) {
+func generateAccessKey(tenant string, master bool) (keyName cluster.KeyString, key *cluster.AccessKey, err error) {
```

Comment:
Can you not set the master attribute at the call site since it looks like it's not always used?
```go
keyName, masterKey, err := generateAccessKey(masterTenantName)
// ...
masterKey.Master = true
```
and the boolean literals are far from readable:
```go
getAccessKey(tenantName, false) // false what?
```